### ISSUE TITLE
deleting "old" bridges with ph cli

### ISSUE THREAD

**Issue by jakew02** [ebaauw/homebridge-hue#574]
Hi,

I didn't realize I still had my Philips bridge v2 still plugged and powered when starting up homebridge-hue for the first time, now on homebridge start I am getting errors from the Philips bridge being unreachable at it's IP (since I've unplugged it from power).

ph discover -v is showing the following:

{
  "deconz4": {
    "192.168.1.6": "00212EFFFF040D7C"
  },
  "deconz4Old": {},
  "deconz6Old": {},
  "meethue": {
    "192.168.1.241": "001788FFFE7FD804"
  },
  "meethueOld": {},
  "upnp": {
    "192.168.1.6": "00212EFFFF040D7C"
  }
}

Basically, I want to just wipe these all out and "start fresh". I tried uninstalling and reinstalling homebridge-hue but it didn't seem to clear anything out.

Is it possible to delete some of these "old" gateways using "ph delete" ?

I wasn't able to find much by searching.

tl;dr I want to delete everythign except "deconz4" (and I guess I need to keep upnp as well?) at 192.168.1.6, everything else should be irrelevant to my setup.

Thanks

**Comment by ebaauw** [comment]
The stale registration of the Hue bridge with the meethue portal should be cleaned automatically after a while.  If you cannot wait for that, disable `nupnp` discovery in config.json or specify only the deCONZ gateway as `host`.  See Wiki for details.

**Comment by jakew02** [comment]
Yes it seems that overnight "meethue" from my above list has forgotten the IP but the previous gateways are still showing. Where are these values stored? I removed the entire persist dir from homelab and grepped for the strings but I can't seem to find them..

**Comment by ebaauw** [comment]
Still showing where?

**Comment by jakew02** [comment]
Still showing via `ph discover -v`

**Comment by ebaauw** [comment]
They’re stored on the meethue portal.
